File: FRONTEND/src/app/core/model/product.ts
export class Product {
  id: number;
  name: string = '';
  description: string = '';
  price: number = 0;
  quantity: number = 0;
  constructor(name: string = '', description: string = '', price: number = 0,
    quantity: number = 0) {
    this.name = name;
    this.description = description;
    this.price = price;
    this.quantity = quantity;
  }
}

File: BACKEND/ApiProjectWeb/Model/Entity/Product.cs
using Model.Entity.GenericObject;

namespace Model.Entity
{
    public class Product : IdentityObject
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
    }
}
